kymatio/scattering2d/frontend/sklearn_frontend.py

```diff
@@ -0,0 +1,15 @@
+from kymatio.frontend.sklearn_frontend import ScatteringTransformerMixin
+
```

Comment:
space?

Reply:
See above (or the rest of the code).

Reply:
Never mind. Could remove this one, sure.

Reply:
Never mind. Could remove this one, sure.